README.md

```diff
@@ -241,3 +242,15 @@ ib-sriov supports the following [CNI's Capabilities / Runtime Configuration](htt
 
 EOF
 ```
+# SR-IOV Network Operator
+[SR-IOV Network Operator](https://github.com/openshift/sriov-network-operator) can manages the SR-IOV interfaces on the nodes e.g. change the number of VFs on the node, it can be used also to change the link type for the interfaces ETH to IB and vice versa, the [network policy example](https://github.com/openshift/sriov-network-operator/blob/master/deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml#L38) shows how to use the operator to change the link type for the given PCI Addresses of PFs.
```

Comment:
>can manages the SR-IOV interfaces on the nodes 

is used to manage SR-IOV interfaces on the node

> it can be used also to change the link type for the interfaces ETH to IB and vice versa

it is also used to change the link type of an interface to Ethernet or Infiniband.

>change the link type for the given PCI Addresses of PFs
change the link type and SR-IOV attributes for a given PCI physical function address